### DISCUSSION TITLE
Can't seem to download anything

### DISCUSSION THREAD

**Discussion by Sandhu-Saabji** [RandomNinjaAtk/arr-scripts#239] [Q&A]
I've setup everything according to the instructions and added the ARL to the config file, there aren't any errors in the logs but I can't seem to download anything

**Comment by RandomNinjaAtk** [comment]
Post logs from the script

**Reply by Sandhu-Saabji** [reply]
All I can find are the logs for lidarr, there aren't any logs being generated under /extended/logs

**Reply by RandomNinjaAtk** [reply]
If you don't have any logs, that indicates the scripts aren't running. Make sure you have followed the installation instructions correctly and configured associated conf file correctly.
